aws-logging-monitoring/monitoring/server-configmap.yaml

```diff
@@ -0,0 +1,645 @@
+      - action: replace
+        source_labels:
+        - __meta_kubernetes_namespace
+        target_label: namespace
+      - action: replace
+        source_labels:
+        - __meta_kubernetes_pod_name
+        target_label: pod
+      - action: keep
+        regex: ^https$
+        source_labels:
+        - __meta_kubernetes_service_annotation_monitoring_gke_io_scheme
+      - action: drop
+        regex: ^(monitoring-ca|seesaw-ca)$
+        source_labels:
+        - __meta_kubernetes_service_annotation_monitoring_gke_io_tls_config
+      scheme: https
+    - job_name: gke-monitoring-io-pods-https-none
+      kubernetes_sd_configs:
+      - role: pod
+      relabel_configs:
+      - action: labelmap
+        regex: __meta_kubernetes_pod_label_(.+)
+      - action: keep
+        regex: true
+        source_labels:
+        - __meta_kubernetes_pod_annotation_monitoring_gke_io_scrape
+      - action: replace
+        regex: (.+)
+        source_labels:
+        - __meta_kubernetes_pod_annotation_monitoring_gke_io_path
+        target_label: __metrics_path__
+      - action: replace
+        regex: ([^:]+)(?::\d+)?;(\d+)
+        replacement: $1:$2
+        source_labels:
+        - __address__
+        - __meta_kubernetes_pod_annotation_monitoring_gke_io_port
+        target_label: __address__
+      - action: replace
+        source_labels:
+        - __meta_kubernetes_namespace
+        target_label: namespace
+      - action: replace
+        source_labels:
+        - __meta_kubernetes_pod_name
+        target_label: pod
+      - action: keep
+        regex: ^https$
+        source_labels:
+        - __meta_kubernetes_pod_annotation_monitoring_gke_io_scheme
+      - action: drop
+        regex: ^(monitoring-ca|seesaw-ca)$
+        source_labels:
+        - __meta_kubernetes_pod_annotation_monitoring_gke_io_tls_config
+      scheme: https
+    - bearer_token_file: /var/run/secrets/kubernetes.io/stackdriver-prometheus-scrape/token
+      job_name: alertmanager
+      kubernetes_sd_configs:
+      - role: endpoints
+      relabel_configs:
+      - action: labelmap
+        regex: __meta_kubernetes_service_label_(.+)
+      - action: replace
+        regex: ([^:]+)(?::\d+)?
+        replacement: $1:9443
+        source_labels:
+        - __address__
+        target_label: __address__
+      - action: keep
+        regex: alertmanager
+        source_labels:
+        - __meta_kubernetes_service_label_k8s_app
+      - action: replace
+        source_labels:
+        - __meta_kubernetes_namespace
+        target_label: namespace
+      - action: replace
+        source_labels:
+        - __meta_kubernetes_pod_name
+        target_label: pod
+      scheme: https
+      tls_config:
+        ca_file: /var/run/secrets/kubernetes.io/stackdriver-prometheus-scrape/ca.crt
+        insecure_skip_verify: true
+    - bearer_token_file: /var/run/secrets/kubernetes.io/stackdriver-prometheus-scrape/token
+      job_name: cadvisor
```

Comment:
somehow the `kubelet` and `cadvisor` metrics are not working in my test. It's probably because our kubelet  does not enable authentication with service account tokens. Ref: https://github.com/prometheus-operator/prometheus-operator/issues/926

A possible solution is to use `schema: http`, remove the `bearer_token_file` and `tls_config`, and add a address relabel to use the `10255` default unsecured port of kubelet:

```
relabel_configs:
      - source_labels: [__address__]
        action: replace
        regex: ([^:]+):.*
        replacement: $1:10255
        target_label: __address__
 ```

Reply:
Nice! Done.